content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,212 @@
+
+   | **Delivery:**
+   | Shipping Product: DPD Home 0-31.5kg
+   | Carrier: DPD
+   | Minimum Weight: 0.00
+   | Maximum Weight: 31.50
+
+   Countries: Austria, Belgium, Bosnia and Herzegovina, Bulgaria, Croatia, Czech Republic,
+   Denmark, Estonia, Finland, France, Germany, Greece, Hungary, Iceland, Ireland, Italy, Latvia,
+   Liechtenstein, Lithuania, Luxembourg, Monaco, Netherlands, Norway, Poland, Portugal, Romania,
+   Serbia, Slovakia, Slovenia, Spain, Sweden, Switzerland
+
+   | **Return:**
+   | Return Shipping Product: DPD Return 0-20kg
+   | Return Carrier: DPD
+   | Return Minimum Weight: 0.00
+   | Return Minimum Weight: 20.00
+   | Return Countries: Belgium, Netherlands
+
+.. image:: sendcloud_shipping/sendcloud-example.png
+   :align: center
+   :alt: Example of shipping products configured in Odoo.
+
+.. tip::
+   Sendcloud does not provide test keys when a company tests the sending of a package in Odoo. This
+   means if a package is created, the configured Sendcloud account will be charged, unless the
+   associated package is canceled within 24 hours of creation.
+
+   Odoo has built an extra layer of protection into test environments. If the shipping method is
+   used to create a label, then the labels are immediately canceled after the creation. This occurs
+   automatically. The test and production environments can toggled back and forth from the
+   :guilabel:`Smart Buttons`.
+
+Generate a label with Sendcloud
+-------------------------------
+
+When creating a quotation in Odoo, add shipping and a :guilabel:`Sendcloud shipping product`. Then,
+:guilabel:`Validate` the delivery. Shipping label documents are automatically generated in the
+chatter, which include the following:
+
+#. :guilabel:`Shipping label(s)` (depending on the number of packages).
+#. :guilabel:`Return label(s)` (if the Sendcloud connector is configured for returns).
+#. :guilabel:`Customs document(s)` (should the destination country require them).
+
+Additionally, the tracking number is now available.
+
+.. important::
+   When return labels are created, Sendcloud will automatically charge the configured Sendcloud
+   account.
+
+FAQ
+===
+
+Shipment is too heavy
+---------------------
+
+If the shipment is too heavy for the Sendcloud service that is configured, then the weight is split
+to simulate multiple packages. Products will need to be put in different :guilabel:`Packages` to
+:guilabel:`Validate` the transfer and generate labels.
+
+:guilabel:`Rules` can also be set up in Sendcloud to use other shipping methods when the weight is
+too heavy. However, it should be noted that these rules will not apply to the shipping price
+calculation on the sales order.
+
+When using a personal carrier contract
+--------------------------------------
+
+If using a personal carrier contract in Sendcloud, and the user finds the price is not accurately
+reflected when creating a quotation in Odoo, the pricing needs to be updated in Sendcloud.
```

Comment:
Suggestion to replace `the user` with an if/then statement that uses price as the subject.
```suggestion
When using a personal carrier contract in Sendcloud, if the the price is not accurately reflected
when creating a quotation in Odoo, then the pricing information needs to be updated in Sendcloud.
```